src/components/About/About.jsx

```diff
@@ -0,0 +1,18 @@
+import './About.css';
+
+function About({closeModal}) {
+
+    return (
+        <div className="about">
+            <div className="about__container">
+                <button onClick={() => closeModal()} type="button" className='about__close'></button>
```

Comment:
`onClick={() => closeModal()}`

**Could be improved**



No need in the arrow function here


```js
onClick={closeModal}
```